test/product.test.ts

```diff
@@ -6,34 +6,18 @@ import { describe, it } from 'mocha';
 import app from '../src/app';
 import path from 'path';
 import { pagination } from '../src/utils';
+import {headerTokenSeller} from './2FA.tets'
 
 chai.use(chaiHttp);
 const { expect } = chai;
-export let token: string = '';
+
 
 describe('PRODUCT API TEST', () => {
-  before(done => {
-    const user = {
-      email: 'test3@example.com',
-      password: '1111@aa',
-    };
-    chai
-      .request(app)
-      .post('/api/users/login')
-      .send(user)
-      .end((err, res) => {
-        expect(res).to.have.status(200);
-        expect(res.body).to.have.property('data');
-        token = res.body.data;
-        done();
-      });
-  });
   it('User should be authenticated', done => {
```

Comment:
if the test should be required you must fake the login  as part of testing

Reply:
Fabrice in my tests I've logged fake seller user and do some tests to get token that we used in product and category the fake login you are telling me is is done in the 2FA.test.ts and i import token to the product and category